core/model/edition.py

```diff
@@ -49,6 +49,22 @@ class Edition(Base, EditionConstants):
     # in a pinch.
     MAX_FALLBACK_THUMBNAIL_HEIGHT = 500
 
+    # Postgresql doesn't allow indices to exceed 1/3 of a buffer page.
+    # We saw the following error here: https://ebce-lyrasis.atlassian.net/browse/PP-188:
+    #
+    # Index row size 3208 exceeds btree version 4 maximum 2704 for index "ix_editions_author"
+    # DETAIL:  Index row references tuple (48187,9) in relation "editions".
+    # HINT:  Values larger than 1/3 of a buffer page cannot be indexed.
+    #
+    # On rare occasions the author (and sort_author) fields can contain a concatenated list of a
+    # large number of authors which breaks the index and causes failures.  What exactly that threshold is
+    # I am not entirely certain.  It appears that 2704 is the size that broke the 1/3 of a buffer page
+    # limit. However, I'm not entirely sure if the buffer page can always be assumed to be a fixed length.
+    # Also, it is not clear to me what other bytes are included in the index row.  Therefore, I have chosen
+    # a value that is both large enough to ensure little or no information is hidden from the end user in most
+    # cases and small enough that we never have to worry about this causing indexing failures in the future.
```

Comment:
It seems like we should be able to validate this with a test case. We could use the constant below to construct an author of the appropriate length.

Reply:
Also, what are we losing when truncate here? If we have a long list of authors, it seems like we might lose the ability to display any beyond the truncation length. If that's the case, it seems like we should do the least harm.